ko/news/_posts/2015-10-13-fukuoka-ruby-award-2016.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: news_post
+title: "2016 후쿠오카 루비 어워드 경진대회 - 참가작을 마츠 씨가 심사합니다"
+author: "Fukuoka Ruby"
+translator: yous
+date: 2015-10-13 17:00:00 +0900
+lang: ko
+---
+
+루비 지지자 여러분께,
+
+일본 후쿠오카 정부와 "마츠" 마츠모토가 당신을 다음 루비 경진대회에 초대합니다.
+흥미로운 루비 프로그램을 개발했다면 지원해보시기 바랍니다.
+
+2016 후쿠오카 루비 어워드 경진대회
+- 대상 - 1백만 엔!
+
+참가 신청 마감: 2015년 11월 27일
+
+마츠와 패널리스트들이 후쿠오카 경진대회의 우승자를 뽑을 것입니다. 후쿠오카
+경진대회의 대상 상금은 1백만 엔입니다. Rhomobile(미국)과 APEC 기후
+센터(대한민국)는 대상을 수상한 바 있습니다.
+
+http://myfukuoka.com/category/news/ruby-news/
+
+경진대회에 등록할 프로그램을 루비로만 작성할 필요는 없지만, 루비만의 독특한
+특징을 잘 활용해야 합니다.
+
+프로젝트는 지난 12개월 이내에 개발되거나 완성된 것이어야만 참가할 수 있습니다.
+더 많은 정보를 보거나 등록하려면 아래 후쿠오카 웹 사이트를 방문해 주세요.
+
+http://www.digitalfukuoka.jp/events/84
+또는
+http://myfukuoka.com/events/2016-fukuoka-ruby-award-guidelines-for-applicants/
+
+http://www.digitalfukuoka.jp/uploads/event_detail/file/210/RubyAward_ApplicationForm_2016.doc
+
+<award@f-ruby.com>으로 신청서를 제출해주세요.
+
+올해엔 다음과 같은 특별한 상을 준비했습니다.
+
+Engine Yard 상:
```

Comment:
Oh, didn't noticed this commit. https://github.com/digitalfukuoka/www.ruby-lang.org/commit/de00266466ca602b4cbaa3c4d5bc140689f3de51